### DISCUSSION TITLE
Limit NUMERIC DIGITS

### DISCUSSION THREAD

**Discussion by adesutherland** [adesutherland/CMS-370-BREXX#60] [Ideas]
I am considering if we should limit NUMERIC DIGITS to 15 - because this is the maximum that BREXX/370 can handle (and even then there will be some bugs to fix to ensure that int32's are promoted to doubles (or that we get rid of int32s all together).

What do you guys think - and what should be an appropriate error message for a NUMERIC DIGITS x where x>15?

Adrian

**Comment by rvjansen** [comment]
I'd say something like:
Error XX.X:  DIGITS value must be a positive number not greater than 15; found "XX".

**Reply by mgrossmann** [reply]
I believe limiting to 15 is the right way. We did it nearly the same way, if I‘m not wrong.
